### ISSUE TITLE
New upgrades (cinnamon 3.0.1) break LMDE 2

### ISSUE THREAD

**Issue by koshak83** [linuxmint/Cinnamon#5261]
Ok: so I saw after installing new updates related to the new Cinnamon 3.0.1:
After all updates are installed on a clean system, packs "Cinnamon" and "Nemo" are removed, and the new versions does not installed. They have to be install manually, or when, on next reboot- you can't enter the system.
https://itmages.ru/image/view/4197687/c56ad330
I also observed that after manually install missing packets and reboot, "login melody" is played twice, with an interval of ~5-10 seconds.

**Comment by surenenfiajyan** [comment]
I also had that breakage and having the melody issue too. When I turn melody off it is still played (but once).

**Comment by Sunderland93** [comment]
Yes, i have melody issue too. One after boot system, second - after boot desktop.

**Comment by As13r** [comment]
Hello
I have the same issue. I'm tryng to do a clean install and everything works fine until I do the updates. This break cinnamon and nemo, and after restart my computer I cant log in in the graphic session anymore. I tried the apt-get install cinnamon from the comand line and didn't solve the problem

**Comment by brice1024** [comment]
Hello,

I have the same issue on an installed LMDE. I installed xfce instead and could access my session.
LMDE is a good concept (better than ubuntu based distros),

Hope the issue will be fixed.

**Comment by darethehair** [comment]
Yes, I had no problems on my LMDE PCs, except for my netbook, on which this occurred.  Installed 'cinnamon' successfully to get the desktop back again, but then I had 'cinnamon-desktop' upgrade request waiting for me when I did get in.  Also got the double chimes.
